packages/gowitness.vm/gowitness.vm.nuspec

```
@@ -2,13 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>gowitness.vm</id>
-    <version>3.0.5.20250219</version>
+    <version>3.0.5.20250506</version>
     <authors>sensepost</authors>
-    <description>Website screenshot utility written in Golang, that uses Chrome Headless to generate screenshots of web interfaces using the command line, with a handy report viewer to process results.</description>
+    <description>gowitness is a website screenshot utility using Chrome Headless to generate screenshots of web interfaces using the command line and a report viewer to process results.</description>
```

Remove "web screenshot" duplication:
```suggestion
    <description>gowitness uses Chrome Headless for website screenshots and provides a built-in report viewer.</description>
```